app/controllers/Management.scala

```diff
@@ -19,7 +19,7 @@ class Management(val controllerComponents: ControllerComponents, val config:Rest
       |Hostname: ${config.hostName}
       |Composer domains: ${config.allStacks.map(_.composerPrefix).mkString(", ")}
       |Snapshots buckets: ${config.sourceStacks.map(_.snapshotBucket).mkString(", ")}
-      |Credentials: ${AWS.creds}
```

Comment:
I'm not sure that this is doing anything useful now.  At some point, `creds` was a case class, that would have displayed the access key but not the secret key (https://github.com/guardian/flexible-restorer/commit/0e71c71efd9f96968a2c6305f7d8252501b553ab)

But now it just displays the class name and hashCode, so could probably be removed